.gitignore

```diff
@@ -1,6 +1,5 @@
 .DS_STORE
 tmp/
-config.yaml
```

Comment:
Please revert, It might include sensitive data and each one of us use it differently.
I saw that you might added it due to the tests config. so just make sure it won't include also config.yaml that we have on root folder. Or change the name of the file in the tests.

Reply:
done. Renamed the tests configs to `toolsets.yaml`